addons/account/wizard/account_payment_register.py

```diff
@@ -839,9 +865,9 @@ def _compute_qr_code(self):
     def _compute_duplicate_moves(self):
         for wizard in self:
             if wizard.can_edit_wizard:
-                wizard.duplicate_move_ids = self._fetch_duplicate_reference()
+                wizard.duplicate_payment_ids = self._fetch_duplicate_reference().get(0, self.env['account.payment'])
             else:
-                wizard.duplicate_move_ids = self.env['account.move']
+                wizard.duplicate_payment_ids = self.env['account.payment']
```

Comment:
Probably wasn't an issue isn't very likely to be one because it's a wizard but calling it on `self` inside a loop while the `_fetch_duplicate_reference` method below itself can't handle more than one record could one day in very specific condition be problematic.

It would be nice to correct that in another PR.